### ISSUE TITLE
Pixiv: Domain-frontening possible?

### ISSUE THREAD

**Issue by PYUDNG** [tachiyomiorg/tachiyomi-extensions#16388]
### Source name

Pixiv

### Source language

Multi-language

### Describe your suggested feature

Pixiv has been blocked by GFW and cannot be accessed in Mainland-China, while [Domain-frontening](https://en.wikipedia.org/wiki/Domain_fronting) is a possible way to bypass ISP's hostname blacklist. Is it possible to support Domain-frontening for Pixiv?

### Other details

May be [pixez-flutter](https://github.com/Notsfsssf/pixez-flutter) helps with the development, which has Domain-frontening for pixiv.net implemented already.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.14.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by vetleledaal** [comment]
Relevant commit https://github.com/Ehviewer-Overhauled/Ehviewer/commit/2d20da70b8922e0b9988b007c31d970217691cb3
